src/main/java/seedu/address/model/deliverable/util/TimeEventComparator.java

```diff
@@ -0,0 +1,21 @@
+package seedu.address.model.deliverable.util;
+
+import java.time.LocalDateTime;
+import java.util.Comparator;
+
+import seedu.address.model.event.TimeEvent;
+
+/**
+ * Represents a Comparator for Deliverable which sorts by Deadline.
+ */
+public class TimeEventComparator implements Comparator<TimeEvent> {
```

Comment:
Move src/main/java/seedu/address/model/meeting/util/TimeEventComparator.java to src/main/java/seedu/address/model/util/TimeEventComparator.java and use the same class